README.md

```diff
@@ -22,9 +22,20 @@ model_path = cached_path("https://github.com/PathwayCommons/pathway-abstract-cla
 # Load model
 model = ktrain.load_predictor(model_path)
 
-# Predict Example 
+# Predict Examples 
+
+# Trivial 
 prediction=model.predict("Article Title".strip() + ' [SEP] ' + "Article Abstract".strip())
 print(prediction)
```

Comment:
Hmmm not 100% married to this but think it might be worth having a super basic example just to emphasize the function and relative simplicity of the model.predict() call? Maybe I should drop the strips though? With that said, if you feel strongly I should drop this I'm definitely willing to

Reply:
It just seems like duplication to me as you do this exact thing below with real title and abstract text.